prebid/Prebid.js
# AdGeneration,Kumma,Platformio: s3 http domain migration

## Type of change


## Description of change



❓ What
  Updated Amazon S3 Path as the current s3 HTTP domain is getting [deprecated](https://aws.amazon.com/blogs/aws/amazon-s3-path-deprecation-plan-the-rest-of-the-story/)


🔨 Test

* Verify new s3 URL is able to fetch the data as usual.
* Verify any functional impact


## Other information
